File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        String[] str = s.split(" ");
        for(int i = 0 ; i < str.length; i++){
            if(Integer.parseInt(str[i]) > max)
                max = Integer.parseInt(str[i]);
            if(Integer.parseInt(str[i]) < min)
                min = Integer.parseInt(str[i]);
        }
        return String.valueOf(min)+" "+String.valueOf(max);
    }
}

File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = []
    maximum = float('-inf')  # Initialize with negative infinity
    minimum = float('inf')   # Initialize with positive infinity
    
    for i in s.split():  # Split input string by space
        if i:  # Check if the string is not empty
            num = int(i)  # Convert string to integer
            if num >= maximum:
                maximum = num
            if num <= minimum:
                minimum = num
            
    answer.append(str(minimum))  # Convert minimum to string before appending
    answer.append(" ")
    answer.append(str(maximum))  # Convert maximum to string before appending
    
    return ''.join(answer)  # Join the answer list into a single string


